operator/pkg/apis/cassandra/v1beta1/cassandradatacenter_types.go

```diff
@@ -241,6 +244,21 @@ func (dc *CassandraDatacenter) GetRacks() []Rack {
 	}}
 }
 
+// ServiceConfig defines additional service configurations. Note, user can override values set by cass-operator, doing so could break cass-operator functionality.
+type ServiceConfig struct {
+	DatacenterService     ServiceConfigAdditions `json:"dcService,omitempty"`
```

Comment:
Should these be pointers? I ask because I wonder if it would make it easier to determine if the user sets the property, particularly when dealing with nested fields.

Reply:
No, it works fine as it is. I would still need to check the separately the Labels and Annotations.